core-js/src/main/javascript/cdf/dashboard/Dashboard.notifications.js

```diff
@@ -31,37 +31,53 @@ define([
   Dashboard.implement(/** @lends cdf.dashboard.Dashboard# */{
 
     /**
-     * Inits the notification.
+     * @summary Dashboard error codes.
+     * @description Contains the default dashboard error codes.
      *
-     * @private
+     * @type {Object}
+     * @property {string} QUERY_TIMEOUT="Query timeout reached" - Query timeout error message.
+     * @property {string} COMPONENT_ERROR="Error processing component" - Error processing component error message.
+     * @constant
+     * @protected
      */
-    _initNotifications: function() {
-
-      /**
-       * Property with the registered error codes. By default, the QUERY_TIMEOUT and COMPONENT_ERROR code
-       * are registered and assigned a specific error message.
-       *
-       * @type {Object}
-       * @name cdf.dashboard.Dashboard#ERROR_CODES
-       */
-      this.ERROR_CODES = {
-        'QUERY_TIMEOUT': {
-          msg: "Query timeout reached"
-        },
-        "COMPONENT_ERROR": {
-          msg: "Error processing component"
-        }
-      };
+    ERROR_CODES: {
+      "QUERY_TIMEOUT": {msg: "Query timeout reached"},
+      "COMPONENT_ERROR": {msg: "Error processing component"}
     },
 
     /**
-     * Sets additional {@link http://malsup.com/jquery/block/|jQuery.blockUI} options based on the defaults already set.
+     * @summary Initializes the dashboard notifications.
+     * @description Initializes the dashboard notifications.
      *
-     * @param {Object}  options             The options to configure.
-     * @param {string}  options.message     The message or HTML to display in the UI overlay.
-     * @param {Object}  options.css         A json that accepts valid css key/value pairs.
-     * @param {Object}  options.overlayCSS  A json that accepts valid css key/value pairs for the UI overlay.
-     * @param {boolean} options.showOverlay Allows you to show or hide the UI overlay.
+     * @private
+     */
+    _initNotifications: function() {},
```

Comment:
This module follows the same pattern as all other Dashboard.*.js so I left this private method here. It was only initialising the ERROR_CODES constant during dashboard instantiation. I think it makes more sense to have the constant initialised only one time when the modules are loaded, rather than every time a dashboard is instantiated.
Do you want me to revert this change?